roles/confluent.zookeeper/tasks/main.yml

```diff
@@ -18,6 +18,10 @@
   yum:
     name: "{{item}}{{confluent_package_redhat_suffix}}"
     state: latest
+  register: install_zookeeper_package_result
+  until: install_zookeeper_package_result is success
+  retries: 5
+  delay: 90
```

Comment:
As suggested by sneha, can we (now or later) explore the exponential backoff for delay duration?